s/elements/stick/element.ts

```diff
@@ -18,6 +18,9 @@ export class NubStick extends MagicElement {
 	@property({type: String, reflect: true})
 	name: string = "1"
 
+	@property({type: String, reflect: true})
+	transform: string = ""
+
```

Comment:
this `transform` property, and associated logic and styles, should be moved into `stickpad` -- the stickpad should be responsible for positioning/transforming the stick within its own rectangle -- whereas the stick should be "unaware" of its surroundings.

basically, the simple `nub-stick` should not have any logic about anything "outside" of its own concerns.

so why does the `nub-stick` have a `transform` property? well, that's because it's currently concerned with and tangled up with the needs of the stickpad -- but that a bad antipattern, we're failing the principal of "separation of concerns" here.

you should re-architect this so that the stickpad has a little logic as possible, and hopefully no special indication that it might be a part of a stickpad.

in some sense, we want a `nub-stickpad should use and control a nub-stick` -- but in a way, the nub-stick should not be "aware" about whether it's being used in a stickpad or not.

you could imagine we iterate on this whole mobile control system, and produce some new alternatives to a stickpad -- if each new iteration has to add new properties and tangled up logic to the nub-stick, the nub-stick would grow very complicated quickly.

ideally, the `nub-stickpad` would render a stick like this:

```js
// stickpad render
return html`
  <nub-stick
    style="${this.#transform}">
  </nub-stick>
`
```

and the stickpad could maybe have some css styles something like
```css
/* stickpad css */

:host {
  position: relative;
}

nub-stick {
  position: absolute;
}
```